install.sh

```diff
@@ -88,6 +85,96 @@ execute_installation() {
         setup_shared_hook_repositories
         echo # For visual separation
     fi
+
+    thank_you
+}
+
+############################################################
+# Apply legacy fixes if appropriate.
+#
+# Returns:  0
+############################################################
+apply_legacy_fixes() {
+
+    # if not installed this will be empty
+    INSTALLED_VERSION=$(git hooks version 2>/dev/null | grep "Version:" | sed -E "s/^Version: //")
+
+    # Updating from this version needs a single install fixture
+    LEGACY_FIXTURE_SINGLE_INSTALL=$(echo "$INSTALLED_VERSION 2004.272234-8af36c" | awk "{ print (\$1 <= \$2) }")
+
+    if [ "$LEGACY_FIXTURE_SINGLE_INSTALL" = "1" ] &&
+        LEGACY_FIXTURE_SINGLE_INSTALL="true"; then
```

Comment:
I dont want bool variables with "0" or "1", I set it to "true" and always test on its "true" value later...

Reply:
Then why not put the `="true"` after the `then` instead of in the `if`? That would make it a lot more readable. Also, maybe just use two different variable names?

Reply:
Rewrite it, looks ugly.